web/src/features/map-controls/ThemeSelector.tsx

```diff
@@ -30,12 +31,22 @@ export default function ThemeSelector() {
   return (
     <MapOptionSelector
       trigger={
-        <MapButton
-          icon={<BsMoonStars size={14} style={{ strokeWidth: '0.2' }} />}
-          tooltipText={__('tooltips.changeTheme')}
-          dataTestId="theme-selector-open-button"
-        />
+        isMobile ? (
+          <Button
+            icon={<BsMoonStars size={14} style={{ strokeWidth: '0.2' }} />}
+            className="my-0"
+          >
+            {__('tooltips.changeTheme')}
+          </Button>
+        ) : (
+          <MapButton
+            icon={<BsMoonStars size={14} style={{ strokeWidth: '0.2' }} />}
+            tooltipText={__('tooltips.changeTheme')}
+            dataTestId="theme-selector-open-button"
+          />
+        )
```

Comment:
This code 🙂

Reply:
I think it will end up being more complicated, they have different margin, icons, testId and tooltips